flink-cdc-composer/src/main/java/org/apache/flink/cdc/composer/PipelineExecution.java

```diff
@@ -27,10 +29,18 @@ public interface PipelineExecution {
     class ExecutionInfo {
         private final String id;
         private final String description;
+        private final JobClient jobClient;
 
         public ExecutionInfo(String id, String description) {
             this.id = id;
             this.description = description;
+            this.jobClient = null;
```

Comment:
I'd try to avoid `null` usage as much as possible to avoid potential NPEs.  It does look like the `jobClient` is available in all the places that construct `ExecutionInfo`. So can we just change the constructor? If not, please add `@Nullable` wherever is applicable.